tests/observability/metrics/conftest.py

```diff
@@ -1088,3 +1078,64 @@ def vnic_info_from_vm_or_vmi(request, running_metric_vm):
 @pytest.fixture()
 def allocatable_nodes(nodes):
     return [node for node in nodes if node.instance.status.allocatable.memory != "0"]
+
+
+@pytest.fixture()
+def windows_vmi_domain_total_memory_bytes_metric_value_from_prometheus(prometheus, windows_vm_for_test):
+    return get_vmi_memory_domain_metric_value_from_prometheus(
+        prometheus=prometheus,
+        vmi_name=windows_vm_for_test.vmi.name,
+        query=KUBEVIRT_VMI_MEMORY_DOMAIN_BYTE,
+    )
+
+
+@pytest.fixture()
+def updated_dommemstat_windows(windows_vm_for_test):
+    run_ssh_commands(
+        host=windows_vm_for_test.ssh_exec,
+        commands=shlex.split("wsl stress-ng --vm 1 --vm-bytes 512M --vm-populate --timeout 600s *> output.log &"),
```

Comment:
1) double check that stress-ng does not exit when the ssh console is closed - even when you run it as a daemon it still can exit.
in VIRT we are using `nohup` for example:
https://github.com/RedHatQE/openshift-virtualization-tests/blob/main/tests/virt/utils.py#L175

2)  `*> output.log`   -- do you really need to save the output?